src/app/pages/viewing/messages/messages.component.html

```diff
@@ -1,351 +1,379 @@
-<!-- Perform a check to see if the number of messages received is 0 -->
 <div
-  *ngIf="messageList.length === 0; else showMessages"
-  class="m-5"
-  id="no-messages"
+  *ngIf="isLoading; else messages"
+  class="loading-empty-state-container d-flex justify-content-center"
 >
-  No messages...
-  <markdown emoji>:sob:</markdown>
+  <div class="spinner-border" role="status"></div>
 </div>
 
-<!-- If the ngIf above fails, this ngTemplate will show -->
-<ng-template #showMessages>
-  <!-- Toggle switch for message viewing styles -->
-  <div class="row my-3">
-    <div class="col-12">
-      <button
-        (click)="toggleViewStyle()"
-        *ngIf="!viewService.isCarouselView"
-        class="btn-sm btn-primary shadow"
-      >
-        Switch to Booklet
-      </button>
-      <button
-        (click)="toggleViewStyle()"
-        *ngIf="viewService.isCarouselView"
-        class="btn-sm btn-primary shadow"
-      >
-        Switch to Cards
-      </button>
+<ng-template #messages>
+  <!-- Perform a check to see if the number of messages received is 0 -->
+  <div
+    *ngIf="messageList.length === 0; else showMessages"
+    class="loading-empty-state-container d-flex justify-content-center"
+  >
+    <div class="empty-state-container px-5">
+      <img
+        alt="No messages"
+        class="empty-state-img"
+        src="../../../assets/images/EmptyMessages.svg"
+      />
+      <h1 class="empty-state-header pt-2">No messages yet</h1>
+      <p class="empty-state-body pt-2">
+        Go to your
+        <a routerLink="/my-virtrolio" routerLinkActive="navbar-nav-active"
+          >virtrolio</a
```

Comment:
I think this makes more sense because the target page is `my-virtrolio`
```suggestion
        Go to
        <a routerLink="/my-virtrolio" routerLinkActive="navbar-nav-active"
          >your virtrolio</a
```